### DISCUSSION TITLE
Compatibility issue with notch-screen iPhones

### DISCUSSION THREAD

**Discussion by jwjoel** [adityatelange/hugo-PaperMod#1226] [Q&A] [answered]
Hi,

I've noticed a colour mismatch between the notch area and the main body on iPhones with notch displays while using the PaperMod Hugo theme. I'm looking to synchronize these colours for better consistency. Any guidance on how to make this change within the theme would be greatly appreciated.

Thanks in advance for any help on this.

![IMG_B01BB1E4175E-1](https://github.com/adityatelange/hugo-PaperMod/assets/25562443/00318774-707e-43fb-9a45-98f4c58de5e1)

**Comment by c2tz** [comment, marked as answer]
Hello, unfortunately I don't think a solution exists because the notch color is determined by the operating system and device settings, there's no solution to change it via CSS code or browser features. I think the best thing to do is to create a meta tag with theme-color to create a sort of continuity in relation to the notch color. This will work with Safari or Chrome, but not Firefox. [here is the documentation](https://developer.mozilla.org/fr/docs/Web/HTML/Element/meta/name/theme-color)
